ansible/roles/snactor/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+---
+- name: Install required packages for snactor on Fedora, if necessary
+  block:
+    - name: Install dnf-plugins-core on Fedora to enable using copr repositories
+      become: true
+      dnf:
+        name: dnf-plugins-core
+        state: present
+
+    - name: Enable leapp repository on Fedora
+      become: true
+      raw: "dnf copr enable -y {{ snactor.copr.repo }}"
+      register: coprenable
+      changed_when: "coprenable.rc != 0"
+
+    - name: Uninstall old version of snactor on Fedora, if necessary
+      become: true
+      dnf:
+        name: "{{ snactor.package }}"
+        state: absent
+
+    - name: Get snactor build dependencies on Fedora
+      raw: "dnf repoquery -C --arch src --requires {{ snactor.package }} 2> /dev/null"
```

Comment:
I am not sure if this is good approach. Why you decided to go this way rather than yum/dnf builddep?